.travis.yml

```diff
@@ -70,6 +70,9 @@ notifications:
       - "irc.freenode.org#gnocchi"
 
 before_deploy:
+  - python3 -m virtualenv ~/deploy-env
+  - source ~/deploy-env/bin/activate
+  - python --version
   - pip install --user --upgrade pip
   - pip install --user --upgrade six
```

Comment:
just use
```suggestion
  - python3 -m pip install --user --upgrade pip
  - python3 -m pip install --user --upgrade six
```
no?

Reply:
This is to ensure the pypi dpl is executed with python3, see https://github.com/travis-ci/dpl/blob/master/lib/dpl/providers/pypi.rb

since it's in a venv it should catch that pip is in python3 already